draft-ietf-quic-load-balancers.md

```diff
@@ -878,9 +878,9 @@ trips in the connection.
 * While the client is using DCIDs it generated, some exposed fields (IP address,
 UDP port, client-generated destination Connection ID) remain constant for all
 packets sent on the same connection. While this document does not update the
```

Comment:
```suggestion
packets sent on the same connection.

While this document does not update the
```

I say that because I think that this statement applies to the entire list.